src/content/app/tools/blast/components/blast-sequence-alignment/blastSequenceAlignmentHelper.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const ALIGNMENT_LINE_LENGTH = 60;
+
+type Params = {
```

Comment:
This is same as `BlastSequenceAlignmentInput`. Can we import and use that instead?

Reply:
Yeah, I was planning to do just that, and then forgot 🙂

Reply:
✅